Public/Src/Cache/ContentStore/Hashing/ContentHasher.cs

```diff
@@ -387,10 +385,9 @@ private Task<bool> PushHashBufferAsync(byte[] buffer, int offset, int count)
 
             private void TransformBlock(byte[] buffer, int offset, int count, byte[] outputBuffer, int outputOffset)
             {
-                var start = _sw.ElapsedTicks;
+                Stopwatch sw = Stopwatch.StartNew();
                 _hashAlgorithm.TransformBlock(buffer, offset, count, outputBuffer, outputOffset);
-                var elapsedTicks = _sw.ElapsedTicks - start;
-                Interlocked.Add(ref _ticksSpentHashing, elapsedTicks);
+                Interlocked.Add(ref _ticksSpentHashing, sw.ElapsedTicks);

```

Comment:
And, btw, why do you think that Stopwatch is not threadsafe?

Reply:
Discussed offline. `timeSpentHashing` is sometimes listed with the exact same elapsed time as neighboring copies. This is particularly strange when the overall copy time is less than the hashing time.

After our discussion, we agreed that we don't know what has happened here, but there's something fishy about `ElapsedTicks` (`1/Frequency` vs `Elapsed`'s count of 100 nanosecond periods) and we're willing to combine behind the more commonly used `Elapsed` field.